libs/pages/logs/environment/src/lib/ui/pod-logs/pod-logs.tsx

```diff
@@ -103,7 +103,11 @@ export function PodLogs({
                 <div className="mr-2.5 w-4">
                   <StatusChip status={currentPod?.state} />
                 </div>
-                <p className="mr-5 truncate text-xs font-medium text-neutral-100">{data.pod_name}</p>
+                <p className="mr-5 truncate text-xs font-medium text-neutral-100" title={data.pod_name}>
+                  {data.pod_name && data.pod_name.length > 23
+                    ? `${data.pod_name?.substring(0, 10)}...${data.pod_name?.slice(-10)}`
```

Comment:
We have an old helper to apply it `trimId` maybe can we custom it ?

Reply:
Here we go :
https://github.com/Qovery/console/pull/1460/commits/b9f04552d041d10401685d7717bb210b987fbbac